### DISCUSSION TITLE
Jaeger micronaut GRPC

### DISCUSSION THREAD

**Discussion by lucaspaixao-dev** [micronaut-projects/micronaut-data#1560] [Q&A]
Hello,

I'm creating a backend service using micronaut and grpc, and I want to integrate with jaeger. Looking on the internet, I have not found any site to help me. The official documentation doesn't have so many examples.
Can anyone help me?
Thanks!

**Comment by dstepanov** [comment]
Now sure if it's possible right now but with OpenTelemetry tracing it should be possible [tracing-opentelemetry-grpc](https://github.com/micronaut-projects/micronaut-tracing/tree/master/tracing-opentelemetry-grpc) after it's released.
Maybe @burtbeckwith will know more.
